### PR TITLE
Add national footnote as data migration

### PR DESCRIPTION
**Pull request by theharq**
Add data migration to add new national footnote

#### Notes

This PR assumes that the last footnote ID is `013`, so this PR sets the ID to `014`  and adds the new footnote description as `-218`

### CODE REVIEW

**Comment by theharq** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;sHjoVQ8n&#x2F;1248-tariff16-add-in-a-new-uk-national-footnote"> [TARIFF16] Add in a new UK (national) footnote</a>

File: db/data_migrations/20160920123015_add_national_footnote.rb [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+        fdp.footnote_description_period_sid = -217
+        fdp.footnote_type_id = FOOTNOTE_TYPE_ID
+        fdp.footnote_id = FOOTNOTE_ID
+        fdp.validity_start_date = VALIDITY_START_DATE
+        fdp.national = true
+        fdp.operation_date = nil # as it came from initial import
+      }.save
+
+      FootnoteDescription.new { |fd|
+        fd.footnote_description_period_sid = -217
```
Comment (matthewford) [inline db/data_migrations/20160920123015_add_national_footnote.rb]: should be incremented e.g. -218

File: db/data_migrations/20160920123015_add_national_footnote.rb [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+      Footnote.new { |f|
+        f.footnote_type_id = FOOTNOTE_TYPE_ID
+        f.footnote_id = FOOTNOTE_ID
+        f.national = true
+        f.validity_start_date = VALIDITY_START_DATE
+        f.operation_date = nil # as it came from initial import
+      }.save
+
+      FootnoteDescriptionPeriod.new { |fdp|
+        fdp.footnote_description_period_sid = -217
```
Comment (matthewford) [inline db/data_migrations/20160920123015_add_national_footnote.rb]: -218 not to clash with existing period

File: db/data_migrations/20160920123015_add_national_footnote.rb [hunk 3]
```diff
@@ -0,0 +1,362 @@
+TradeTariffBackend::DataMigrator.migration do
+  name "Add national footnote to Taric's XXXXX"
+
+  FOOTNOTE_TYPE_ID = "XX"
+  FOOTNOTE_ID = "XXX"
+  FOOTNOTE_DESCRIPTION = "Where the free rate of duty has been applied due to inclusion of 'Multi-component integrated circuits' (MCOs) document code Y035 (no status or reference required) must be declared at item level in box 44."
+
+
+  TARIC_FOOTNOTE_TYPE_ID = "CD"
```
Comment (matthewford) [inline db/data_migrations/20160920123015_add_national_footnote.rb]: Not sure if we need this as its only a CHIEF footnote

Comment (matthewford) [inline db/data_migrations/20160920123015_add_national_footnote.rb]: yeah the old migration was adding footnotes where there was an existing taric migration, which is why this and code relating to this was here - this one we are just adding in a footnote directly to commodity codes.

**Comment by theharq** [comment]
PR updated

**Pull request merged**